lib/tasks/send_presentation_notifications.rake

```diff
@@ -0,0 +1,7 @@
+require 'rake'
+
+task send_presentation_notifications: :environment do
```

Comment:
Sry man that's my bad. I should have tested this more in depth. Made the missing var change.